token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -69,22 +73,112 @@ pub enum MetadataInstruction {
     Transfer(TransferArgs),
     // Updates the metadata of an asset
     Update(UpdateArgs),
-    // Authorizes the use of a token
-    UseAsset(UseAssetArgs),
+    // Provide utility operations on a token
+    Utility(UtilityArgs),
     // Verifies creator/collection for an asset
     Verify(VerifyArgs),
 }
 ```
 
-Each of these instructions will use versioned `*Args` structs to facilitate future updates, and in turn, not require additional instructions.
+Each of these instructions will use versioned `*Args` structs to facilitate future updates, and in turn, not require additional instructions. Operations supported under each instruction are as follows:
+
+- [ ] `Burn`
+- [X] `Create`
+    * [X] Creation of Programmable Non-Fungibles tokens (pNFT)
+    * [X] Creation of Non-Fungibles tokens (NFT)
+    * [X] Creation of Fungible Assets (*semi-fungible tokens*)
+    * [X] Creation of Fungible Tokens (*fungible tokens*)
+- [ ] `Delegate`
+    * [ ] Creation of `Authority` delegates
+    * [X] Creation of `Collection` delegates
+    * [X] Creation of `Sale` delegates
+    * [X] Creation of `Transfer` delegates
+    * [X] Creation of `Update` delegates
+    * [ ] Creation of `Use` delegates
+    * [X] Creation of `Utility` delegates
+- [ ] `Migrate`
+- [X] `Mint`
+    * [X] Mint Programmable Non-Fungibles tokens (pNFT)
+    * [X] Mint of Non-Fungibles tokens (NFT)
+    * [X] Mint Fungible Assets (*semi-fungible tokens*)
+    * [X] Mint Fungible Tokens (*fungible tokens*)
+- [ ] `Print`
+    * [ ] Print of editions
+- [ ] `Revoke`
+    * [ ] Revoke of `Authority` delegates
+    * [X] Revoke of `Collection` delegates
+    * [X] Revoke of `Sale` delegates
+    * [X] Revoke of `Transfer` delegates
+    * [X] Revoke of `Update` delegates
+    * [ ] Revoke of `Use` delegates
+    * [X] Revoke of `Utility` delegates
+- [ ] `Transfer`
+    * [X] wallet-to-wallet transfers
+    * [X] wallet-to-program transfers
+    * [X] program-to-wallet transfers
+- [X] `Update`
+    * [X] Update metadata details for Programmable Non-Fungibles
+    * [X] Update metadata details for Non-Fungibles
```

Comment:
I'd probably add `FungibleAssets` and `Fungibles` to stay consistent with the other instructions. Otherwise, it looks like the `Update` instruction is only for non-fungibles.

Reply:
Thanks - added both `FungibleAssets` and `Fungibles`.